qargparse.py

```diff
@@ -91,6 +91,10 @@ def addArgument(self, name, type=None, default=None, **kwargs):
             tuple: Enum,
         }.get(type, type)
 
+        # Get default value from QArgument class if not provided
+        if default is None:
```

Comment:
Hm, this seems a bit suspicious. Could this not get handled in the constructor of `Argument`, like the other `kwargs`?

Reply:
Yeah, good point ! Should be fixed in a056c5c.